code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -201,6 +203,24 @@
 	hud_list[IMPTRACK_HUD]    = new /image/hud_overlay('icons/mob/hud.dmi', src, "hudblank")
 	hud_list[SPECIALROLE_HUD] = new /image/hud_overlay('icons/mob/hud.dmi', src, "hudblank")
 
+/mob/living/silicon/robot/proc/update_access()
+	if (emagged || malf_AI_module || crisis)
+		id_card.access = get_all_station_access() + access_synth // Give full station access
+		return
+
+	id_card = new id_card_type()
+
+	if (module)
```

Comment:
During Skull's reviews it was a standard to have spacing after conditional, loop statements just like in languages like Java, C#, C, C++. Did that standard get reverted to opposite?

Reply:
most code is written without spaces. it's also more compact.

Reply:
> most code is written without spaces. it's also more compact.

The old ss13 code yes. But that is why people had cleanup's and automatic formatters that would change it every time they touch a file. It doesn't make sense to use wrong standard just because lots of code is already written in it.

Reply:
Nevermind, looking back at my old PRs seems like it was without spacing. I must have confused it with different codebase. I will change these